app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = []
+    for i in people:
+        new_list.append(Person(i['name'], i['age']))
+
+    for x in people:
+        if 'wife' in x.keys():
+            if x['wife'] is not None:
```

Comment:
use `and` operator instead of nested `if`s

Reply:
done